ChangeLog

```diff
@@ -14,6 +24,7 @@
 	* man/BermudanSwaption.Rmd: Modified to reflect DiscountCurve
 	option and example updated
 	
+>>>>>>> eddelbuettel/master
```

Comment:
I am going to disentangle your paragraph a little.  Note that I cannot speak to the windows desktop product, so I may focus on the command-line.  There are some nice GUIs out there, I am sure suitable tutorials exist.

> 1) Fork master (can be anywhere, so github fork ok, right?

Yes. Forks are always remote so you do it at GitHub (GH for now) via the web.

Before you fork again (see below) rename your existing fork into rquantlib-previous or -v1 or something.

> 2) clone it (did this originally via Studio project open of my github fork)

Yes.  'git clone .....address you get from the project page....'  create a local repo.

> 3) write code, check in with local commit, check it with R CMD check

Correct.  RStudio is good for all of this, sadly it does not do branches well.  If you just work in your master it does not matter so much.  I like branches a lot.

> 4) Run Travis locally (how do I do this?)

You can't.  There are other CI systems that could. Not worth it.  Run `R CMD check` a lot.  Run Travis on your fork.

> 5) Happy with 4, push to fork on github

Correct.  Which will trigger Travis on your fork.  This is important.   You want this to work before you send a PR.

> 6) Create pull checking files are correct (is there a way to delete files from a pull request?)

Correct.  You can alter / remove / add / ... files in a pull request by via git -- ie `git rm ....` followed by commit and push removes.

> So what if my repo is out of sync? Do I delete current forks and clones and redo steps 1& 2? 

You update and merge.  This is basic git stuff.   Work through some tutorials.

This is not hard.  There are only you and me here, and I work linearly.  If you end up in a mess it is all you, and only you can get yourself out.

> When I did the sync, I thought it was creating a copy of Master, but I suspect I had some changes on my side that it worked into the repo. How do I get a clean new Master?

I would start with a new fork, after first renaming (assuming you want to keep it) the remote existing fork.   If you trust your local copies you can also just delete the existing fork and start over.